convert2rhel/subscription.py

```diff
@@ -413,3 +415,44 @@ def check_needed_repos_availability(repo_ids_needed):
             all_repos_avail = False
     if all_repos_avail:
         loggerinst.info("Needed RHEL repos are available.")
+
+
+def download_subscription_manager_packages():
+    loggerinst = logging.getLogger(__name__)
+
+    _CENTOS_CONTRIB_REPO = '[contrib]\n' \
+                           'name=CentOS-$releasever - Contrib\n' \
+                           'mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib\n' \
+                           'gpgcheck=0\n' \
+                           'enabled=1\n' \
+                           'protect=0\n'
+
+    _TEMP_REPOFILE_DIR = '/tmp/convert2rhel/centos_contrib.repo'
+
+    version = utils.parse_version_number(system_info.system_release_file_content)
```

Comment:
If they use a system we don't support, e.g. major version 5, we'll get no logs at all. Add a log at the end of the function for when we don't support a system.